public/pages/feed.html

```diff
@@ -24,174 +26,195 @@
         <a class="navbar-brand visible-brand-inDesktop" href="#">
         </a>
         <b>
-        <ul class="nav">
-          <li class="nav-item">
-            <a class="nav-link navbar-link" href="/public/pages/about.html"> QUEM SOMOS </a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link navbar-link" href="/public/pages/feed.html"> FEED </a>
-          </li>
-          <li class="nav-item dropdown">
-            <a class="nav-link navbar-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button"
-              aria-expanded="false">ATIVIDADES</a>
-            <ul class="dropdown-menu">
-              <li>
-                <a class="dropdown-item" href="/public/pages/games/">Jogos</a>
-              </li>
-              <li>
-                <a class="dropdown-item" href="/public/pages/games/licoes_leitura.html">
-                  Lições de Leitura
-                </a>
-              </li>
-              <li>
-                <a class="dropdown-item" href="/public/pages/games/leitura_assistida.html">
-                  Leitura Assistida
-                </a>
-              </li>
-            </ul>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link navbar-link" href="/public/pages/contato.html"> CONTATO </a>
-          </li>
-        </ul>
+          <ul class="nav">
+            <li class="nav-item">
+              <a class="nav-link navbar-link" href="../pages/about.html"> QUEM SOMOS </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link navbar-link" href="../pages/feed.html"> FEED </a>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link navbar-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button"
+                aria-expanded="false">ATIVIDADES</a>
+              <ul class="dropdown-menu">
+                <li>
+                  <a class="dropdown-item" href="../games/index.html">Jogos</a>
+                </li>
+                <li>
+                  <a class="dropdown-item" href="../games/licoes_leitura.html">
+                    Lições de Leitura
+                  </a>
+                </li>
+                <li>
+                  <a class="dropdown-item" href="../games/leitura_assistida.html">
+                    Leitura Assistida
+                  </a>
+                </li>
+              </ul>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link navbar-link" href="../pages/contato.html"> CONTATO </a>
+            </li>
+          </ul>
         </b>
-        <a class="btn btn-outline-primary btn-signup" href="/public/pages/cadastro/cadastro.html"> Cadastre-se </a>
+        <a class="btn btn-outline-primary btn-signup" href="../pages/cadastro/cadastro.html"> Cadastre-se </a>

```

Comment:
Sugestão de correção:

```suggestion
        <a class="btn btn-outline-primary btn-signup" href="./cadastro/cadastro.html"> Cadastre-se </a>
```